src/test/java/seedu/address/testutil/EditPropertyDescriptorBuilder.java

```diff
@@ -41,6 +41,8 @@ public EditPropertyDescriptorBuilder(Property property) {
         descriptor.setPhone(property.getPhone());
         descriptor.setEmail(property.getEmail());
         descriptor.setTags(property.getTags());
+        descriptor.setTagsToAdd(null);
+        descriptor.setTagsToDelete(null);
```

Comment:
Should the descriptor be passed `null` here when property has a getter for `tagsToAdd` and `tagsToDelete`?